package/MDAnalysis/core/Selection.py

```diff
@@ -198,29 +267,29 @@ def _apply_KDTree(self, group):
         for atom in sel.positions:
             kdtree.search(atom, self.cutoff)
             found_indices.append(kdtree.get_indices())
-        logger.debug("Found indices are {}".format(found_indices))
+        #logger.debug("Found indices are {0}".format(found_indices))
         # These are the indices from SYS that were seen when
         # probing with SEL
         unique_idx = np.unique(np.concatenate(found_indices))
-        logger.debug("Unique indices are {}".format(unique_idx))
+        #logger.debug("Unique indices are {0}".format(unique_idx))
         return unique(sys[unique_idx.astype(np.int32)])
 
     def _apply_distmat(self, group):
-        logger.debug("In Around Distmat")
+        #logger.debug("In Around Distmat")
         sel = self.sel.apply(group)
-        logger.debug("Sel is {}".format(sel))
+        #logger.debug("Sel is {0}".format(sel))
         sys = group[~np.in1d(group.indices, sel.indices)]
-        logger.debug("Sys is {}".format(sys))
+        #logger.debug("Sys is {0}".format(sys))
 
         box = group.dimensions if self.periodic else None
         dist = distances.distance_array(
             sys.positions, sel.positions, box)
-        logger.debug("dist has shape {}".format(dist.shape))
-        logger.debug("dist is {}".format(dist))
+        #logger.debug("dist has shape {0}".format(dist.shape))
+        #logger.debug("dist is {0}".format(dist))
 
         mask = (dist <= self.cutoff).any(axis=1)
 
-        logger.debug("mask has shape {}".format(mask.shape))
+        #logger.debug("mask has shape {0}".format(mask.shape))
```

Comment:
Logging at DEBUG level will show up in the log file if you use `mda.start_logger()`.  Something that is clearly only used in development (such as the ones here) should be eventually removed.
If you think that it will eventually help you to diagnose a problem that a user might have then leave it in.